skrub/_table_vectorizer.py

```diff
@@ -146,10 +150,72 @@ def _replace_missing_in_cat_col(ser: pd.Series, value: str = "missing") -> pd.Se
     return ser
 
 
-Transformer = TransformerMixin | Literal["drop", "remainder", "passthrough"]
+def _clone_if_default(transformer, DEFAULT_TRANSFORMER):
+    return clone(transformer) if transformer is DEFAULT_TRANSFORMER else transformer
+
+
+def _clone_during_fit(transformer, remainder, n_jobs):
```

Comment:
as `_clone_during_fit`, `_check_specific_transformers` and `_propagate_n_jobs` are called by `TableVectorizer` methods, with attributes of `TableVectorizer` as their arguments, why did you choose to make them module-level functions rather than methods?

Reply:
ok I was just curious I don't have a strong opinion